config/database.yml

```diff
@@ -3,6 +3,7 @@ default: &default
   encoding: utf8
   username: collections_pub
   password: collections_publisher
+  host: <%= ENV['DATABASE_HOST'] || 'localhost' %>
```

Comment:
Can we use database_url instead of database_host? see: http://edgeguides.rubyonrails.org/configuring.html#configuring-a-database

Otherwise we'd only be able to configure one MySQL database in the docker-compose rather than having one per application